labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -32,8 +32,37 @@ object CollectionExercise01 {
    * Case 3: so it is okay if you want to just give up
    *
    */
+  def repl(c:Char):Char = {
+    Map('a' -> 'y',
+    'b' -> 'h',
+    'c' -> 'e',
+    'd' -> 's',
+    'e' -> 'o',
+    'f' -> 'c',
+    'g' -> 'v',
+    'h' -> 'x',
+    'i' -> 'd',
+    'j' -> 'u',
+    'k' -> 'i',
+    'l' -> 'g',
+    'm' -> 'l',
+    'n' -> 'b',
+    'o' -> 'k',
+    'p' -> 'r',
+    'q' -> 'z',
+    'r' -> 't',
+    's' -> 'n',
+    't' -> 'w',
+    'u' -> 'j',
+    'v' -> 'p',
+    'w' -> 'f',
+    'x' -> 'm',
+    'y' -> 'a',
+    'z' -> 'q',
+    ' ' -> ' ')(c)
+  }
```

Comment:
It makes complete sense to do things this way, though it's a lot of work on your end to make this map based on the sample strings provided. Maybe you could consider automating the map creation by zipping each character of the example inputs and outputs.